plugins/default/lib/nagios.py

```diff
@@ -3,3 +3,12 @@
 WARN=1
 CRIT=2
 UNKNOWN=3
+
+def status_code_to_label(code):
+	return {
```

Comment:
Could we run `autopep8` to reformat all Python files?

We should probably document it too :)

https://pypi.python.org/pypi/autopep8